format_implementing.md

```diff
@@ -0,0 +1,92 @@
+
+### Sanitization
+
+It's recommend to whitelist tags and attributes based on conventions from a host. For example, if the host you're integrating your launcher with only uses `<p>` tags, only allow `<p>` tags to be rendered. It's a better approach that minimizes any potential risk.
+
+Make sure to sanitize all attributes. For example, XSS attacks can use the `onerror` attribute to run JavaScript code after loading an invalid image. Only allow the attributes that are required.
+
+### Frequently Asked Questions
+
+**Q: I've received HTML in a field that doesn't allow for it in the specification. Should I parse it anyway?**  
+A: No. Do not attempt to parse HTML outside of a field that it's allowed in. If you receive a string, `<p>My Mod</p>` in the `name` field, render it exactly as written, including the HTML tags. Don't attempt to strip HTML from a value.
+
+**Q: I've received Markdown. Should I parse it?**  
+A: Markdown is not allowed in the specification. You shouldn't attempt to parse it. Hosts should be providing HTML, not Markdown.
+
+**Q: I don't want to render HTML at all. Should I just strip it out?**  
+A: If for whatever reason you do not want to render HTML, then yes, the best method is to remove the tags. Keep in mind the consequences of not rendering HTML, as users may lose helpful and useful features such as images and video embeds.
+
+---
+
+## Implementing Frameworks
+
+Frameworks also exist as MODIP Format Projects. It's important for launchers to understand the Project metadata and install frameworks as needed.
+
+Framework IDs MUST be prefixed with `framework-`, so launchers can tell if a project is a Framework.
+For frameworks with the installation method set to `versionJsonInstall`, launchers should read the version JSON metadata specified in a file with `rel` set to `versionJson`. An example of a modloader using `versionJsonInstall` would be Fabric.
+
+Launchers may deviate from or completely ignore the value set in the installation method for the version.
+
+The `runForgeInstaller` installation method exists to tell launchers that this contains a Forge installer. Launchers may choose to use their own method of installation if they would like.
+
+---
+
+## Implementing Dependencies
+
+When a user downloads a project using a launcher, it's important to download its required dependencies. Launchers should combine the version dependencies with the file dependencies in order to create a full dependency list.
+
+For the purposes of this section, the term "the host" refers to the service a launcher is using in order to download projects, such as Diluv or Modrinth.
+
+If a dependency's ID is `minecraft`, then it's the required Minecraft version. Implementation of where to obtain Minecraft metadata is up to the developer of a launcher. Launchers may download metadata from the host, from Mojang directly, or from another service.
```

Comment:
I think there shouldn't be special-case IDs, or at least, handling them should be optional. If `minecraft` is a valid dependency then we have to provide a project JSON entry for `minecraft`. Or if `minecraft` is its own special thing, then it should have its own special field.